src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -24,7 +24,9 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
+        String username = req.getParameter("username");
```

Comment:
`<input type="text" name="login" form="driver" required>`
can you get `username`?